theme/_layouts/index.html

```diff
@@ -36,6 +36,7 @@ <h1>Get Started</h1>
                                 </a>
                             </div>
                             <img id="aws-logo" class="hidden-xs" src="{% if jekyll.environment == 'production' %}{{ site.amplify.baseurl }}{% endif %}/images/Logos/aws_logo.png" />
+			    <div class="below-registration-button callout callout--action">Note: It may take up to 24 hours for your AWS account to activate. You'll get an error creating an Amplify project if it is not activated. Visit <a href="https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/" target="_blank">this page</a> for information on getting your account activated sooner.</div>
```

Comment:
1. Fix indentation
1. Run this by @undefobj to confirm placement. As a "getting started" page, it doesn't really seem appropriate to be putting troubleshooting information here. That said, if we're getting a lot of customer feedback about the account activation error, it may make sense to hoist it higher. My instinct is that this probably needs to be closer to the step-by-step instructions.

Reply:
I haven't heard any reports of this yet, though it does occasionally happen. I would suggest not merging this change unless the issue starts to become more prevalent.

Reply:
I counted 11 reports of it between these two issues:

- https://github.com/aws-amplify/amplify-cli/issues/191
- https://github.com/aws-amplify/amplify-cli/issues/972

Reply:
I think 8 reports over a year+ is a minor thing. This is ok to leave out for now.

Reply:
Ok sounds good - closing out